Clockify.Net/Models/TimeOff/TimeOffEndpointRequest.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+
+namespace Clockify.Net.Models.TimeOff
+{
+    public class TimeOffEndpointRequest
+    {
+        public DateTime End { get; set; }
+        public DateTime Start { get; set; }
+        public int Page { get; set; }
+        public int PageSize { get; set; }
+        public IEnumerable<TimeOffRequestStatusType> Statuses { get; set; }
+        public IEnumerable<string> UserGroups { get; set; }
+        public IEnumerable<string> Users { get; set; }
```

Comment:
We should probably be auto-initializing these to empty lists so that no one is getting null references
```suggestion
        public IEnumerable<TimeOffRequestStatusType> Statuses { get; set; } = new List<TimeOffRequestStatusType>();
        public IEnumerable<string> UserGroups { get; set; } = new List<string>();
        public IEnumerable<string> Users { get; set; }= new List<string>();
```